package.json

```diff
@@ -12,22 +12,16 @@
   },
   "license": "MIT",
   "dependencies": {
```

Comment:
❓ Maybe this needs to be rebased. I don't think we want to remove these dependencies.